src/App.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import UserProfile from './components/UserProfile.jsx';
+import NotFoundPage from './containers/NotFoundPage';
+
+class App extends Component {
+  render() {
+    return (
+      <>
```

Comment:
Good design @TheDrizzyWay, I will like to please verify if this line has any function or it is a typo

Reply:
It's shorthand for a react fragment.

Reply:
https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html